panther_bringup/launch/bringup.launch.py

```diff
@@ -51,9 +51,10 @@ def generate_launch_description():
         choices=["True", "true", "False", "false"],
     )
 
+    robot_model = EnvironmentVariable("ROBOT_MODEL", default_value="panther")
```

Comment:
@KmakD please, check to correct eeprom env name

Reply:
```suggestion
    robot_model_env = EnvironmentVariable("ROBOT_MODEL", default_value="PTH")
    robot_model = "lynx" if robot_model == "LNX" else "panther"
```
Values are PTH/LNX. If you need full names you can use syntax in second line

Reply:
I still do not understand why we use unintuitive shortcut instead of full model name.

Reply:
This naming is more production like. Launch argument can be a full name, just the env variable should be remapped.

Reply:
its changed, but still don't like it